src/pages/CreateDesign/components/Footer/index.tsx

```diff
@@ -25,62 +26,67 @@ const Footer = (): React.ReactElement => {
   const [currentStep, setCurrentStep] = useRecoilState(currentStepAtom);
   const {
     name,
+    designType,
+    patternType,
     stitches,
     rows,
+    size,
+    needle,
+    yarn,
+    extra,
+    description,
+    targetLevel,
+    coverImageUrl,
+    techniques,
+  } = useRecoilValue(currentDesignInputAtom);
+  const {
     totalLength,
     sleeveLength,
     shoulderWidth,
     bottomWidth,
     armholeDepth,
-    needle,
-    yarn,
-    extra,
-    designType,
-    patternType,
-  } = useRecoilValue(currentDesignInputAtom);
+  } = size;
   const editorState = useRecoilValue(editorStateAtom);
 
+  const { mutate } = usePost({
+    pathname: '/design/',
```

Comment:
p2: trailing slash 제거 하는걸로 통일하고 있어서 여기두 빼는게 좋을 것 같네용

```suggestion
    pathname: '/design',
```